UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
I just understand why I'm getting these warnings: I have a space in a folder name: `/path/with space/to/SwiftGen`. Dropping this space solves the issues, I just have warnings about an `!` left (thanks to your update).

So, I'm gonna change my folder name now 😆